src/components/SiderMenu/BaseMenu.tsx

```diff
@@ -65,21 +79,22 @@ export default class BaseMenu extends PureComponent {
           }
           key={item.path}
         >
-          {this.getNavMenuItems(item.children)}
+        {/* To Verify: why parent isnt supplied here. funct def: getNavMenuItems(menusData, parent) */}
+        {getNavMenuItems(item.children)}
         </SubMenu>
       )
     }
-    return <Menu.Item key={item.path}>{this.getMenuItemPath(item)}</Menu.Item>
-  };
+    return <Menu.Item key={item.path}>{getMenuItemPath(item)}</Menu.Item>
+  }
```

Comment:
I think we can move this function to module scope as well.

Reply:
done.